File: IT/Funzioni/Esercizi/C++/esercizio5.cpp
/* file esercizio5.cpp */

#include <iostream>

using std::cout;
using std::endl;
using std::boolalpha;

bool comp(int a, int b) {
	return a > b;
}

int main() 
{
	int a = 5;
	int b = 10;

	cout << a << " > " << b << "? " << boolalpha << comp(a, b) << endl;
	cout << b << " > " << a << "? " << boolalpha << comp(b, a) << endl;
}

File: IT/Funzioni/Esercizi/Java/Esercizio5.java
/* file Esercizio5.java */

public class Esercizio5 {
	static boolean comp(int a, int b) {
		return a > b;
	}

	public static void main(String [] args) {
		int a = 5;
		int b = 10;

		System.out.println(a + " > " + b + "? " + comp(a, b));
		System.out.println(b + " > " + a + "? " + comp(b, a));
	}
}

